ds701_book/21-Networks-I.qmd

```diff
@@ -253,422 +387,409 @@ nx.draw_networkx(DG,
 
-The second way to measure clustering coefficient is the mean of the ratios:
+---
+
+The second way to measure the clustering coefficient is the mean of the ratios, i.e.,
     
-$$ C^{(2)} = \frac{1}{n} \sum_i \frac{\text{number of triangles that include }i}{\text{number of pairs of neigbors of }i} $$
+$$ 
+C^{(2)} = \frac{1}{n} \sum_i \frac{\text{number of triangles that include node } i}{\text{number of pairs of neighbors of node }i},
+$$
+
+where $n$ is the total number of nodes.
+
+This is the probability that neighbors are connected for a __random node__.
 
-This is the probability that neighbors are connected for a __random node.__
+In the previous example, $C^{(2)} = \frac{1}{5} \left(1 + 1 + \frac{1}{6}\right) = \frac{13}{30} = 0.433.$
 
-$$ C^{(2)} = \frac{1}{5} (1 + 1 + \frac{1}{6}) = \frac{13}{30} = 0.433$$
+---
 
 What is the clustering coefficient in a $G(n, p)$ random graph?
 
-This is easy: the probability that two of your neighbors are connected is the same as the probability that __any__ two nodes are connected.
-    
-In other words, $C^{(1)} = C^{(2)} = p$.
+The probability that two of your neighbors are connected is the same as the probability that __any__ two nodes are connected, i.e., $C^{(1)} = C^{(2)} = p$.
 
-__Real World Graphs Tend to Show Large Amounts of Clustering.__
+## Real World Graphs 
 
-In practice one finds that real world graphs show strong clustering.
+In practice, real world graphs show strong clustering.
 
-For example, consider a social network.  Your friends are much more likely to be themselves friends than two randomly chosen people.
+For example, consider a social network. Your friends are much more likely to be themselves friends than two randomly chosen people.
 
-<center>
-    
-<img src="figs/L21-clustering-coefs.png" alt="Figure" width="800px">
-    
-</center>
+![](figs/L21-clustering-coefs.png){fig-align=center width=800px}
 
 ## Clustering and Path Length: Small Worlds
 
-The strong presence of clustering in networks leads to a question: how long is a typical shortest path between nodes?
+The strong presence of clustering in networks leads to a question. 
+
+How long is a typical shortest path between nodes?
     
-The average shortest path length between nodes is one way to measure a network's "diameter."
+The average shortest path length between nodes is one way to measure a network's *diameter*.
 
-Let's consider a highly clustered graph:
+---
 
-<center>
-    
-<img src="figs/L21-high-clustering.png" alt="Figure" width="300px">
-    
-</center>
+Let's consider a highly clustered graph.
 
-In this graph, each node has four neighbors, and of the 6 pairs of neighbors, 3 are connected.
+![](figs/L21-high-clustering.png){fig-align=center width=300px}
 
-So the clustering coefficient is 0.5, quite high.
+In this graph, each node has 4 neighbor. Of the 6 pairs of neighbors, 3 are connected, yielding a clustering coefficient of 0.5. This is quite high.
```

Comment:
What are the 6 pairs?

Reply:
If a node has 4 connected neighbors, then their are 6 possible pairs of neighbors (4 choose 2).

Reply:
I added a statement to mention that the 6 pairs are based off of 4 choose 2.